dulwich/repo.py

```diff
@@ -328,7 +328,13 @@ def _determine_file_mode(self) -> bool:
         """
         raise NotImplementedError(self._determine_file_mode)
 
-    def _init_files(self, bare: bool) -> None:
+    def _determine_symlinks(self) -> bool:
```

Comment:
The function is defined at line 1766.  Is this stub needed?

Reply:
This is just documentation for any subclasses, which will need to provide an implementation.